docs/source/docker/advanced/datasource_predictions.rst

```diff
@@ -13,8 +13,8 @@ add their predictions to the datasource.
 If you already have labels instead of predictions, you can treat them
 just as predictions and upload them the same way.
 
-.. warning:: Note that using predictions requires a minimum 
-    docker of version 2.2. You can check your docker version by running the 
+.. note:: Note that the object level features require a minimum 
+    Lightly Worker of version 2.2. You can check your docker version by running the 
```

Comment:
you can chekc your docker version -> you can check the version of Lightly Worker by running the sanity check..